web/js/codeworld_shared.js

```diff
@@ -234,15 +234,38 @@ function renderDeclaration(decl, keyword, keywordData, maxLen) {
     wordElem.appendChild(document.createTextNode(keyword));
     decl.appendChild(wordElem);
 
+    let leftover = "";
     if (keywordData.symbolEnd < keywordData.declaration.length) {
-        let leftover = keywordData.declaration.slice(keywordData.symbolEnd).replace(
+        leftover = keywordData.declaration.slice(keywordData.symbolEnd).replace(
             /\s+/g, ' ');
         if (keywordData.symbolEnd + leftover.length > maxLen && leftover.length >
             3) {
             leftover = `${leftover.slice(0, maxLen - 3 - keywordData.symbolEnd) 
             }...`;
         }
-        decl.appendChild(document.createTextNode(leftover));
+    }
+    if (argIndex >= 0) {
+        if (!leftover.length) {
+            leftover = " :: Could not decipher arguments";
```

Comment:
I'm assuming this message is there for debugging?  In production, it would be better to not show the type than to complain about not being able to figure it out.

Reply:
Okay. i will remove this.